fixtures/varytec/giga-bar-frost-pix-8-rgb.json

```diff
@@ -0,0 +1,244 @@
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "helpWanted": "At which DMX values is strobe disabled?"
+      }
+    },
+    "Multi-function": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed",
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [8, 15],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in red",
+          "colors": ["#ff2600"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [16, 23],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in red and green",
+          "colors": ["#ff2600", "#00f900"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [24, 31],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in green",
+          "colors": ["#00f900"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [32, 39],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in blue and green",
+          "colors": ["#0433ff", "#00f900"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [40, 47],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in blue",
+          "colors": ["#0433ff"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [48, 55],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in blue and red",
+          "colors": ["#0433ff", "#ff2600"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [56, 63],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in red, green and blue",
+          "colors": ["#ffffff"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [64, 231],
+          "type": "Effect",
+          "effectName": "Built-in automatic show programs",
```

Comment:
So there are multiple programs here? Can you tell at which DMX values the effects change?

Reply:
According to the 'Operating mode ‘Built-in automatic show’' section of the manual, there are multiple shows (Pr01 - Pr21) and options to set a 'background colour' and a 'hopping colour'. However, these options only seem available when the fixture is not being DMX controlled.
In DMX 2ch mode, the function of Ch2 changes depending on the value of Ch1. This was difficult behaviour to program in QLC+. I had to use the [Aliases feature](https://www.qlcplus.org/docs/html_en_EN/fixturedefinitioneditor.html#aliases) which was new to me. Ch2 only changes the speed, there is no mention of being able to select a program. I'm therefore not sure if it cycles through Pr01 - Pr21 or just sticks to one. 
I can test the fixture at the weekend and report back.